carmajava/launch/environment.launch

```diff
@@ -33,6 +33,124 @@
   <!-- World Model Updater -->
   <include file="$(find carma_wm_ctrl)/launch/carma_wm_broadcaster.launch"/>
 
-  <!-- TODO Object Tracking and World Model go here -->
+  <!-- Object Tracking and World Model -->
+
+  <!-- vision_darknet_detect -->
+  <group>
+  <include file="$(find vision_darknet_detect)/launch/vision_yolo3_detect.launch">
+  <arg name="gpu_device_id" default="0"/>
+  <arg name="score_threshold" default="0.30"/>
+  <arg name="nms_threshold" default="0.30"/>
+
+  <arg name="network_definition_file" default="$(find vision_darknet_detect)/darknet/cfg/yolov3.cfg"/>
+  <arg name="pretrained_model_file" default="$(find vision_darknet_detect)/darknet/data/yolov3.weights"/>
+  <arg name="names_file" default="$(find vision_darknet_detect)/darknet/cfg/coco.names"/>
+
+  <arg name="camera_id" default="/"/>
+  <arg name="image_src" default="$(optenv CARMA_INTR_NS)/camera/image_raw"/>
+  </include>
+  </group>
+
+<!-- vision_beyond_track -->
+  <group>
+  <include file="$(find vision_beyond_track)/launch/vision_beyond_track.launch">
+  <arg name="camera_info_src" default="$(optenv CARMA_INTR_NS)/camera/camera_info"/>
+  <arg name="objects_topic_src" default="detection/image_detector/objects"/>
+  <arg name="camera_height_calibration_params_file" default="$(find vehicle_calibration_dir)/vision_beyond_track/calibration.yaml"/>
```

Comment:
There is no vehicle_calibration_dir package. You should add an argument to the environment.launch file called vehicle_calibration_dir and pass it in from carma_src.launch where it already exists. See localization.launch for an example.